content/docs/deployment/csmoperator/modules/_index.md

```diff
@@ -10,4 +10,4 @@ The steps include:
 
 1. Deploy the Dell CSM Operator (if it is not already deployed). Please follow the instructions available [here](../../#installation).
 2. Configure any pre-requisite for the desired module(s). See the specific module below for more information
-3. Follow the instructions available [here](../drivers/powerscale.md/#install-driver)) to install the Dell CSI Driver via the CSM Operator. The module section in the ContainerStorageModule CR should be updated to enable the desired module(s). There are [sample manifests](https://github.com/dell/csm-operator/tree/main/samples) provided which can be edited to do an easy installation of the driver along with the module.
+3. Follow the instructions to install the Dell CSI Driver (such as [PowerScale](../drivers/powerscale/#install-driver) or [PowerFlex](../drivers/powerflex/#install-driver)) via the CSM Operator. The module section in the ContainerStorageModule CR should be updated to enable the desired module(s). There are [sample manifests](https://github.com/dell/csm-operator/tree/main/samples) provided which can be edited to do an easy installation of the driver along with the module.
```

Comment:
I see PowerStore is added for 1.6 CSM Operation support. I think we have to add PowerStore here.

Reply:
No. Let this be as an example. I am fixing the link in the main